### DISCUSSION TITLE
POE security Cameras all work in Frigate but send Notifications after version 0.14 help

### DISCUSSION THREAD

**Discussion by vjmaz** [blakeblackshear/frigate#14748] [General]
Hi Team 
       Before the really Kool version (0.14) came out I have been running Frigate Addon in Home assistant or long time using these older 1080P POE TianAnXUN cameras. Also I have ANNKE NVR that has old Analog Cameras which has 18 channels. now I have 7 Analog cams working on the NVR and from the NVR I go to the Frigate via Network using RTSP protocol . 

The NVR also allows POE Cameras to connect to it only Via adding network address and specs in the add POE camera settings . The NVR only has 0ne Network cable it doesn't have extra POE ports ... but I have added 5 POE camera`s and it runs very good .

Now everything before frigate Version 0.14 was awesome working really well on the NVR and on Frigate .. Frigate would send notifications  as needed and all its features worked ...

After 0.14 update everything works but the POE cameras ( TianAnXUN ) capture record via Frigate but don't send Notifications to my phone ... I don't have any other devices only my phone that gets notified ! ...

Now this said I also do get notified on the 7 Analog Cameras and even on a single POE camera that is not the same brand it is a iC See wifi/4G Smart Camera which has cat5 ( LAN) connection which is what I use and it works fine with notifications ? ....

I am not sure were to start checking team which logs or were , and I m not familiar with MQTT team .

If anyone as any ideas how I can get these POE TianAnXUN cameras to work would be much appreciated please as I have 8 of them ... 

( things I have tried ) 
Ordinally the TianAnXUN cameras connected directly to Frigate via RTSP Protocol , so I tried to go via the RTSP protocol from the NVR also  but nothing changed.

Added a complete fresh TianAnXUN camera on different ports and different IP still nothing different .

thank you

**Comment by hawkeye217** [comment]
Frigate's Home Assistant integration does not send notifications natively. Are you using the sgtbatten blueprint for notifications?

**Reply by vjmaz** [reply]
I am hawkeye217 
![image](https://github.com/user-attachments/assets/bdeab2dd-ce70-417f-8b4f-5e0e60d227ee)

**Reply by hawkeye217** [reply]
You'll want to make sure you're using the beta version of the blueprint if you're using Frigate 0.14.

**Reply by vjmaz** [reply]
Ok interesting I will try that later on the week when I get back home, Thank you hawkeye 217 will provide update later :)

**Comment by vjmaz** [comment]
Hawkeye217 That
 worked Thank you very Much 
coffee purchased :) 
![image](https://github.com/user-attachments/assets/eada73f3-e865-4d56-bb9d-5ceae9257a3d)

**Reply by hawkeye217** [reply]
Thanks!
